packages/react/src/components/SimplePagination/SimplePagination.jsx

```diff
@@ -48,56 +60,63 @@ const SimplePagination = ({
   const handlePrev = () => onPage(page - 1);
 
   return (
-    <div className={`${iotPrefix}-simple-pagination-container`}>
-      <span className={`${iotPrefix}-simple-pagination-page-label`} maxpage={maxPage}>
-        {pageText ? `${pageText} ${page}` : pageOfPagesText(page, maxPage)}
-      </span>
-      {maxPage > 1 ? (
-        <>
-          <div
-            className={
-              hasPrev
-                ? `bx--pagination__button bx--pagination__button--backward ${iotPrefix}-addons-simple-pagination-button`
-                : `bx--pagination__button bx--pagination__button--backward ${iotPrefix}-addons-simple-pagination-button-disabled`
-            }
-            role="button"
-            tabIndex={hasPrev ? 0 : -1}
-            onClick={hasPrev ? handlePrev : undefined}
-            onKeyDown={hasPrev ? (evt) => handleEnterKeyDown(evt, handlePrev) : undefined}
-          >
-            <CaretLeft16
-              dir="ltr"
-              description={prevPageText}
+    <div className={`${iotPrefix}--simple-pagination-container`} data-testid={testID}>
```

Comment:
Should these class name changes happen in a minor version? It could potentially create a lot of issues for people using xpath tests.